From 5e902e5e32c672661dfe5677c4a950c4dd409198 Mon Sep 17 00:00:00 2001
From: loqs <loqs@users.noreply.github.com>
Date: Sun, 23 May 2021 20:29:17 +0100
Subject: [PATCH] Fix boost 1.76.0 compatibility

Add missing header boost/mpl/contains.hpp
monero-project/monero/issues/7728
---
 contrib/epee/include/storages/portable_storage.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/contrib/epee/include/storages/portable_storage.h b/contrib/epee/include/storages/portable_storage.h
index 655a2eb12c..60f3e672fb 100644
--- contrib/epee/include/storages/portable_storage.h
+++ contrib/epee/include/storages/portable_storage.h
@@ -31,6 +31,8 @@
 #include "misc_log_ex.h"
 #include "span.h"
 
+#include <boost/mpl/contains.hpp>
+
 namespace epee
 {
   class byte_slice;
